lib/Doctrine/Common/DataFixtures/Exception/CircularReferenceException.php

```diff
@@ -4,8 +4,8 @@
 
 namespace Doctrine\Common\DataFixtures\Exception;
 
-use Doctrine\Common\CommonException;
```

Comment:
That would be a BC break. I could revert that if the remaining changes are good for a minor release.

Reply:
Please do revert it :pray: Also, why did you need to do that in the first place, if you recall?

Reply:
I removed the reference to `Doctrine\Common\CommonException` because this class is also deprecated:

> Doctrine\Common\CommonException is deprecated without replacement.
see: https://www.doctrine-project.org/2018/07/12/common-2-9-and-dbal-2-8-and-orm-2-6-2.html

I'm fine to revert that line and just keep the updated DBAL calls.

Reply:
Reverted.